lib/slack/state.ex

```diff
@@ -53,6 +53,17 @@ defmodule Slack.State do
     update_in(slack, [:groups, channel, :members], &Enum.uniq([user | &1]))
   end
 
+  def update(
+        %{type: "message", subtype: "group_topic", channel: channel, user: user, topic: topic},
+        slack
+      ) do
+    update_in(
+      slack,
+      [:groups, Access.key(channel)],
+      &Map.put(&1, :topic, %{creator: user, last_set: System.system_time(:second), value: topic})
```

Comment:
What's the need/idea behind `last_set`? It doesn't look like we're using it, is the thought that the consumer would use it? If so, what's the use case?

Reply:
It's part of the `topic` map that's there to start, so I just mirrored that.